src/Microsoft.VisualStudio.SDK.Analyzers.Tests/DiagnosticVerifier.cs

```diff
@@ -169,6 +169,7 @@ protected static Project CreateProject(string[] sources, string language = Langu
                 Environment.GetEnvironmentVariable("USERPROFILE"),
                 ".nuget",
                 "packages");
+
```

Comment:
This extra whitespace is the only change in the file. Can you revert it? 
I have another PR #27 coming anyway that changes the line above it and I'd rather avoid an unnecessary merge conflict.